src/fusion-rates-gamow.md

```diff
@@ -0,0 +1,861 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,src//md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.15.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+<a href="https://colab.research.google.com/github/project-ida/nuclear-reactions/blob/master/fusion-rates.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://nbviewer.jupyter.org/github/project-ida/nuclear-reactions/blob/master/fusion-rates.ipynb" target="_parent"><img src="https://nbviewer.jupyter.org/static/img/nav_logo.svg" alt="Open In nbviewer" width="100"/></a>
+
+
+# Fusion rates 01
+
+In this notebook, we'll calculate nuclear fusion rates using the [Gamow model](https://web.archive.org/web/20200504014928/http://web.ihep.su/dbserv/compas/src/gamow28/eng.pdf). We'll first focus our attention the spontaneous fusion of two deuterons in a $\rm D_2$ molecule and then explore what fusion rates we might expect when $\rm D_2$ is absorbed into metals like palladium.
+
+This notebook contains results and methods used for fusion rate plots from [Known mechanisms that increase nuclear fusion rates in the solid state](https://doi.org/10.1088/1367-2630/ad091c).
+
+```python
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+from mpl_toolkits.axes_grid1.inset_locator import inset_axes, mark_inset
+import pandas as pd
+from scipy.integrate import simps, trapz
+pd.set_option('display.max_rows', 300)  # or None to display an unlimited number of rows
+```
+
+```python
+# Constants
+e = 1.602176634e-19 # charge on the electron
+eps0 = 8.85418782e-12 # permitivity of free space
+hbar = 1.054571817e-34
+amu = 1.66053906892e-27 # 1 amu in kg
+a0 = 0.529177210903e-10 # bohr radius in m
+Ry_to_eV = 13.605693122990 # 1 Rydberg in eV
+J_to_eV = 1/1.602176634e-19 # 1 Joule in eV
+eV_to_J = 1.602176634e-19 # 1 eV in Joules
+```
+
+```python
+# Deuterium information
+deuteron_mass = 2.013553212745 * amu
+muDD = ((deuteron_mass*deuteron_mass) / (deuteron_mass+deuteron_mass)) # reduced mass of D2 in kg
+```
+
+## Nuclear fusion
+
+
+Fusion can be described as a two step process:
+1. A quantum tunneling event through a potential barrier, with the barrier defined by the interatomic potential between two nuclei.
+2. A relaxation or decay of the highly clustered nuclei into some ground state or decay products (with the concomitant release of energy)
```

Comment:
I wouldn't say "highly clustered nuclei" here -- are they always clustered? Perhaps "highly excited nuclei" would be more general.